packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -305,6 +305,113 @@ class UiKitView extends StatefulWidget {
   State<UiKitView> createState() => _UiKitViewState();
 }
 
+/// Embeds a Linux GtkWidget in the Widget hierarchy.
+///
+/// {@macro flutter.rendering.RenderLinuxView}
+///
+/// Embedding Linux GtkWidget is an expensive operation and should be avoided when a Flutter
+/// equivalent is possible.
+///
+/// {@macro flutter.widgets.AndroidView.layout}
```

Comment:
Not sure how these macros work but I think there should be new ones for Linux?

Reply:
Docs of macOS and iOS `UIKitView` also include these macros:
https://github.com/flutter/flutter/blob/341984237ec63988fffd69754aebabba92093ff2/packages/flutter/lib/src/widgets/platform_view.dart#L204